### PR TITLE
u-boot: dts : remove usb3.0 related dts configurations

### PR DESCRIPTION
**Pull request by AsuraZeng**
This is a known issue that usb3.0 can not work under u-boot.
The usb3.0 port can only work at usb2.0 mode. So remove the usb3.0
related dts configurations,and disable the usb3.0 serdes signal.

Signed-off-by: Chao Zeng <chao.zeng@siemens.com>

### CODE REVIEW

**Review by BaochengSu — APPROVED** [review]

File: recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Date: Wed, 22 Sep 2021 16:47:57 +0800
+Subject: [PATCH 9/9] u-boot: dts : remove usb3.0 related dts configurations
+
+This is a known issue that usb3.0 can not work under u-boot.
+The usb3.0 port can only work at usb2.0 mode. So remove the usb3.0
+related dts configurations and disable the usb3.0 serdes signal
+
+Signed-off-by: chao zeng <chao.zeng@siemens.com>
+---
+ arch/arm/dts/k3-am65-iot2050-common-pg2.dtsi | 16 +---------------
```
Comment (jan-kiszka) [inline recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch:12]: Again: Patch k3-am65-iot2050-common-u-boot.dtsi or create a corresponding one for only pg2 but do not modify this one beyond includes.

**Review by jan-kiszka — CHANGES_REQUESTED** [review]

File: recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch (hunk 1 above)
Comment (AsuraZeng) [reply recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch]: has already create a new dtsi file to restore this feature.

File: recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+ 	compatible = "siemens,iot2050-basic-pg2", "ti,am654";
+diff --git a/arch/arm/dts/k3-am6548-iot2050-advanced-pg2.dts b/arch/arm/dts/k3-am6548-iot2050-advanced-pg2.dts
+index f00dc86d01..1420ba1041 100644
+--- a/arch/arm/dts/k3-am6548-iot2050-advanced-pg2.dts
++++ b/arch/arm/dts/k3-am6548-iot2050-advanced-pg2.dts
+@@ -17,6 +17,7 @@
+ 
+ #include "k3-am6548-iot2050-advanced-common.dtsi"
+ #include "k3-am65-iot2050-common-pg2.dtsi"
++#include "k3-am65-iot2050-u-boot-pg2.dtsi"
```
Comment (jan-kiszka) [inline recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch:72]: Why not add this to k3-am65-iot2050-common-pg2.dtsi? Then you only need to touch one file.

File: recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch [hunk 3]
```diff
@@ -0,0 +1,78 @@
+From b736741e8b5db849b9c698a8bbdd2f3a3549c714 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Fri, 24 Sep 2021 11:50:29 +0800
+Subject: [PATCH 9/9] u-boot: dts : remove usb3.0 related dts configurations
+
+This is a known issue that usb3.0 can not work under u-boot.
+The usb3.0 port can only work at usb2.0 mode. So remove the usb3.0
+related dts configurations,and disable the usb3.0 serdes signal.
+
```
Comment (jan-kiszka) [inline recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch:9]: This does not describe the effect of this change. I assume it's that the port will fall back to USB 2.0 mode and can thus be used this way.

File: recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch (hunk 2 above)
Comment (AsuraZeng) [reply recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch:72]: the usb dts configuration for pg2 exists in the k3-am65-iot2050-common-pg2.dtsi.
it is simple to remove or modify these node. but you mentioned before in order to keep sync with kernel side, it should not modify this file.  So add a new file for the specific change for the pg2 board.

File: recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
++};
++
++&dwc3_0 {
++	assigned-clock-parents = <&k3_clks 151 4>,	/* set REF_CLK to 20MHz i.e. PER0_PLL/48 */
++					 <&k3_clks 151 9>;	/* set PIPE3_TXB_CLK to CLK_12M_RC/256 (for HS only) */
++	/delete-property/ phys;
++	/delete-property/ phy-names;
++};
++
++&usb0_phy {
```
Comment (jan-kiszka) [inline recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch:47]: As we just learned, this must be usb0 once the common-pg2.dtsi file is fixed.

File: recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch (hunk 2 above)
Comment (jan-kiszka) [reply recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch:72]: OK, ordering matters here: We need to include this after the common pg2 bits.

**Pull request closed without merging**